services/venice-server/src/main/java/com/linkedin/venice/stats/ServerQuotaUsageStats.java

```diff
@@ -65,7 +65,7 @@ public void setTokenBucket(TokenBucket tokenBucket) {
    */
   private Double getReadQuotaUsageRatio() {
     if (tokenBucket == null) {
-      return -1d;
+      return Double.NaN;
```

Comment:
Do we have a test to check if NaN is skipped by metric aggregator?

Reply:
I'm still looking for answers for this one. Also wanted to make sure it doesn't cause any like performance issues for the aggregator.

Reply:
Verified that the average aggregator will ignore NaN values, don't think we can have a meaningful integration test for it tho.